drillsrs/cmd/import_.py

```diff
@@ -1,14 +1,36 @@
 import argparse
 import json
 import sys
+import io
 from typing import IO, Any, Optional
 
 from dateutil.parser import parse as parse_date
+from anki_export import ApkgReader
```

Comment:
I'd like this package to be optional

Reply:
It's now Optional and also lxml package since it only used in that there